dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ClaimedTaskRestRepository.java

```diff
@@ -173,7 +174,12 @@ public ClaimedTaskRest findByItem(@Parameter(value = "uuid", required = true) UU
             if (xmlWFI == null) {
                 return null;
             } else {
-                claimedTask = claimedTaskService.findByWorkflowIdAndEPerson(context, xmlWFI, context.getCurrentUser());
+                if (authorizeService.isAdmin(context)) {
```

Comment:
this change is modifying the goal of the endpoint. I would suggest to introduce a new search method reserved to the administrator to retrieve all the claimed task related to a workflow item.
With the current implementation if a workflow item have two or more claimed task associated and one of them belong to the administrator it is possible that the administrator doesn't get it own claimed task. This would mean that the administrator will be not able to perform its task from the mydspace